src/Language/PureScript/Ide/State.hs

```diff
@@ -194,11 +196,11 @@ populateStage2 :: (Ide m, MonadLogger m) => m ()
 populateStage2 = do
   st <- ideStateVar <$> ask
   duration <- liftIO $ do
-    start <- getCurrentTime
+    start <- getTime Monotonic
     atomically (populateStage2STM st)
```

Comment:
I've little to no experience in haskell benchmarking. It might totally be that lazyness is playing games with me here. I'd expect things inside STM to be evaluated after `atomically`, but that's just me guessing.